skills/partiful/references/auth-output-and-safety.md

```
@@ -0,0 +1,63 @@
+partiful events create --help
+partiful schema events.create
+partiful schema guests.invite
+```
+
+Treat live help and schema output as authoritative when this reference and the installed CLI differ.
+
+## Global Flags
+
+| Flag | Purpose |
+|---|---|
+| `--format <json|table|csv|ndjson>` | Output format; JSON is default |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 2; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)